sympy/matrices/matrices.py

```diff
@@ -853,7 +853,7 @@ def rref(self, iszerofunc=_iszero, simplify=False, pivots=True, normalize_last=T
             act as a pivot.  `lambda x: x.is_zero` is used by default.
         simplify : Function
             A function used to simplify elements when looking for a pivot.
-            By default SymPy's `simplify`is used.
+            By default SymPy's `simplify` is used.
```

Comment:
Single back-ticks are used for LaTeX. You should use double back-ticks for code.